soc/realtek/ec/rts5912/Kconfig.defconfig.rts5912

```diff
@@ -0,0 +1,20 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+#
+
+if SOC_RTS5912
+
+config PINCTRL
+	default y
```

Comment:
please, make sure drivers select this as needed

Reply:
We moved it to board level config